openlibrary/plugins/openlibrary/js/wikipedia_citation_button.js

```diff
@@ -0,0 +1,3 @@
+export default function WikipediaCitationButtonPressed() {
```

Comment:
never capitalize function names unless they are used with the `new` operator.

I'm also curious about the function name. From the JS this doesn't have much to do with a button being pressed, it's bound to a focus on a text area.

I would recommend avoiding thinking about naming by just removing this file and adding the line inline above.

How are you testing this?

Reply:
The wikipedia citation menu opens only after pressing the wikipedia citation button, not all books have this on their page. An example is this book: https://openlibrary.org/books/OL26337700M/The_universe ; scroll down / Ctrl+F wikipedia citation, click on it, a window opens and then if you click on the textarea with the citation text, it should select it as whole (for easier copying)

Reply:
The window that opens should have this: 
```
{{Citation
        |title = The universe
        |publication-date = 1970
        |ol = 26337700M
        |url = https://archive.org/details/universe00otto
        |isbn = 9780262690034
        |oclc = 749851604
        |publisher = MIT Press
        |publication-place = Cambridge, Massachusetts
        |author = Otto Struve
        }}
```

Reply:
Thank you! Super helpful context! Couldn't replicate this locally.